tetris.py

```diff
@@ -336,13 +395,14 @@ def user_input_right(self):
             return
         x, y = self.current_block_center
         self.current_block_center = (x + 1, y)
+        self.main_board.draw_block(self.get_block_shape(), self.block_letter, self.landed_blocks)
```

Comment:
This line of code is copy/pasted into many places. Maybe make a new method that contains it? Something like `def redraw(self): self.main_board....`

Reply:
Maybe `redraw_main_board` would be a better name. Choose something that makes sense to you.